ironfish/src/wallet/wallet.ts

```diff
@@ -984,7 +979,9 @@ export class Wallet {
 
     if (broadcast) {
       await this.addPendingTransaction(transaction)
-      this.memPool.acceptTransaction(transaction)
+      await this.nodeClient.mempool.acceptTransaction({
+        transaction: transaction.serialize().toString('hex'),
+      })
```

Comment:
Should we handle RPC errors on these nodeClient calls? If a request fails because of connectivity the wallet process will crash